shared/lib/wpcom-undocumented/lib/me.js

```diff
@@ -1,6 +1,8 @@
 /**
  * Module dependencies.
  */
+import _Me_ from 'wpcom-unpublished/dist/lib/me';
```

Comment:
What's the reason behind the naming here? Any reason not to simply name it `Me`?